lib/connection/pool.js

```diff
@@ -412,7 +412,7 @@ function messageHandler(self) {
     }
 
     // Log if debug enabled
-    if(self.logger.isDebug()) {
+    if(self.logger.isDebug() || true) {
```

Comment:
I think this will always resolve to true meaning the debug message will be printed.